tools/c7n_left/c7n_left/cli.py

```diff
@@ -20,7 +20,7 @@
 @click.group()
 def cli():
     """Shift Left Policy"""
-    logging.basicConfig(level=logging.DEBUG)
```

Comment:
Should the level be a CLI argument like `--verbose` so we can get more details if needed?

Reply:
yeah I debated that, but there's not really anything additional from a utility log wise atm, definitely open to the option in the future but its mostly noise atm.